scripts/cluster-upgrade.sh

```diff
@@ -410,6 +467,7 @@ teardown_maintenance_window() {
     OCM_NAME=$1
     CD_NAMESPACE=$2
     CD_NAME=$3
+    MAINT_DESC=$4
```

Comment:
Same as above. Does this need to be quoted to prevent extra words being set to `$5`, `$6`, etc?